meta-client/src/test/java/org/apache/helix/metaclient/puppy/AbstractPuppy.java

```diff
@@ -30,8 +30,8 @@ public abstract class AbstractPuppy implements Runnable {
 
   protected MetaClientInterface<String> metaclient;
   protected PuppySpec puppySpec;
-  protected HashMap<String, Integer> eventChangeCounterMap;
-  protected int unhandledErrorCounter;
+  public HashMap<String, Integer> eventChangeCounterMap;
```

Comment:
sorry, may be few dumb questions - why would you make the member public ?

Reply:
Not a dumb question! I intend to have the implementation of Lattice specific puppies in a separate package so for it to be accessed, it would need to be public.